### DISCUSSION TITLE
remap property values in pnp filter using a custom template

### DISCUSSION THREAD

**Discussion by floses-p** [microsoft-search/pnp-modern-search#3431] [Q&A] [answered]
Dear  pnp - community,

is it possible to remap the showing value of mapped properties in the 'PnP - Search Filters' webpart?

I am trying to filter by the property 'SiteTemplate', which indicates the type of Sharepoint Site (Team, Communication, Channel etc.).

The values showing dont make munch sense for standard users, there for i would like to remap the values with something that makes more sense. See screenshot below.

![Snag_1c54990](https://github.com/microsoft-search/pnp-modern-search/assets/152491525/f9d45989-87ab-4bbf-8843-cff8d5bb80e8)

Any help is very appretiated! 

Thank's in advance

**Comment by kasperbolarsen** [comment, marked as answer]
One option could be to create a custom Refiner layout and simply replace the text values?
I have recently used that approach in this blog post ( link to the layout template at the end of the blog post)
https://www.m365thinking.com/post/flags-in-the-pnp-modern-search-filter

**Reply by floses-p** [reply]
Thank you, problem solved :)
